pkg/plugins/policies/meshhttproute/plugin/v1alpha1/testdata/default-meshexternalservice.clusters.golden.yaml

```diff
@@ -2,19 +2,37 @@ resources:
 - name: meshexternalservice_example
   resource:
     '@type': type.googleapis.com/envoy.config.cluster.v3.Cluster
-    dnsLookupFamily: V4_ONLY
-    loadAssignment:
-      clusterName: meshexternalservice_example
-      endpoints:
-      - lbEndpoints:
-        - endpoint:
-            address:
-              socketAddress:
-                address: example.com
-                portValue: 10000
-          loadBalancingWeight: 1
+    edsClusterConfig:
+      edsConfig:
+        ads: {}
+        initialFetchTimeout: 0s
+        resourceApiVersion: V3
     name: meshexternalservice_example
-    type: STRICT_DNS
+    transportSocket:
+      name: envoy.transport_sockets.tls
+      typedConfig:
+        '@type': type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+        commonTlsContext:
+          alpnProtocols:
+          - kuma
+          combinedValidationContext:
+            defaultValidationContext:
+              matchTypedSubjectAltNames:
+              - matcher:
+                  exact: spiffe://default/zone-egress
```

Comment:
Do we have a way to add egress config to a golden file and assert on that? With the current config we know that egress is configured but we don't know if it's configured correctly 🤔

Reply:
It would require a lot of changes. We would need to configure Egress generators and probably everything around the context :/

Reply:
I still think it makes sense to do it - let's open up an issue about this and let's talke it in 2.10.x